frontend/components/EmptyState.vue

```diff
@@ -61,37 +61,37 @@
               v-if="pageType == 'organizations'"
               class="w-full"
               :cta="true"
-              label="components.empty_state.create_organization"
+              label="i18n.components.empty_state.create_organization"
```

Comment:
Bit of Vue learning here, @aasimsyed: What we do here is preppend `label` with `:`, which is short for `v-bind:label` ([see v-bind docs on W3 Schools](https://www.w3schools.com/vue/vue_v-bind.php)). This allows us to pass a variable instead of a string. I'll fix these quickly 😊